### PR TITLE
Add documentation to `ShardCloseCode` and `AuditLogEventType`

### PR DESCRIPTION
**Pull request by mplatypus**
### Summary
Added documentation to `ShardCloseCode` and `AuditLogEventType`

### Checklist
- [x] I have run `nox` and all the pipelines have passed.
- [x] I have made unittests according to the code I have added/modified/deleted.

### Related issues
Nothing related.

### CODE REVIEW

File: hikari/audit_logs.py [hunk 1]
```diff
@@ -293,52 +293,99 @@ class AuditLogEventType(int, enums.Enum):
...
     MESSAGE_BULK_DELETE = 73
+    "Multiple messages were deleted."
     MESSAGE_PIN = 74
+    "Message was pinned to a channel."
     MESSAGE_UNPIN = 75
+    "Message was unpinned from a channel."
     INTEGRATION_CREATE = 80
+    "App was added to server."
     INTEGRATION_UPDATE = 81
+    "App was updated (as an example, its scopes were updated)."
```
Comment (davfsa) [inline hikari/audit_logs.py:362]:
```suggestion
     INTEGRATION_UPDATE = 81
    """An integration was updated.

    For example, its scopes were updated.
    """
```

**Review by davfsa — CHANGES_REQUESTED** [review]
Thanks for the pull request!

Couple of notes:
- Docstrings should begin and end with `"""`, not `"`
- I am not too sure about adding documentation to the shard close codes, but I wouldn't mind it. It seems a bit arbitrary

**Comment by davfsa** [comment]
Also, changelog fragment is missing :)

**Comment by mplatypus** [comment]
@davfsa
> * I am not too sure about adding documentation to the shard close codes, but I wouldn't mind it. It seems a bit arbitrary

What about the 4000-4999? as found here [mdn](https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent/code) it does state that those close codes are private, and therefore unique. I was thinking adding it to all of them just looked nicer imo.

**Comment by davfsa** [comment]
Yeah, that's part of the spec, 4xxx range is for individual use after agreement. It's fine leaving them documented ig.

Would you mind fixing the docstring issue (wrapping with triple quotes instead of single) and I think this is good to merge

**Comment by mplatypus** [comment]
was literally just doing that right now haha.

**Pull request closed without merging**